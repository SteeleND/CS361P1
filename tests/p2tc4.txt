a
a

aea a0a a1a
0
000
011
e
10111
