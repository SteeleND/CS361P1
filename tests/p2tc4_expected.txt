Q = { [a] }
Sigma = { 0 1 }
delta =
                   0         1
       [a]       [a]       [a]
q0 = [a]
F = { [a] }

yes
yes
yes
yes
yes
